src/main/apiClient.ts

```diff
@@ -308,15 +308,13 @@ export default class ApiClient {
         maybeBrowserTab.openUrl(url, () => {}, logError)
         return Promise.resolve()
       } else if (window.cordova.InAppBrowser) {
-        if (window.cordova.platformId === 'ios') {
+        const ref = window.cordova.platformId === 'ios' ?
           // Open a webview (to pass Apple validation tests)
-          window.cordova.InAppBrowser.open(url, '_blank')
-        } else {
+          window.cordova.InAppBrowser.open(url, '_blank') :
           // Open the system browser
           window.cordova.InAppBrowser.open(url, '_system')
-        }
 
-        return Promise.resolve()
+        return Promise.resolve(ref)
```

Comment:
Don't we have to update the function's return type?

Reply:
Yes absolutely, I got you. 
As I just searched for a quick way to unblock me for my app, and due to deadline to put my application in production, in didn't take the time to search more for the type, neither for adding a test case. 

As seen in the documentation (https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-inappbrowser/), 
> The cordova.InAppBrowser.open() function is defined to be a drop-in replacement for the window.open() function

So it should be typed as function for the return type, imo.. Not totally sure about that. 
To avoid regressions, maybe we should just define the return function as ` Promise<void | Function>` , what do you think about that ? 

I updated the PR.